specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/ServerDevOpsAudit.json

```diff
@@ -166,8 +167,59 @@
     }
   },
   "definitions": {
-    "ServerDevOpsAuditPolicyProperties": {
-      "description": "Properties of a server DevOps audit policy.",
+    "SystemData": {
```

Comment:
Any reason not to reference the common-type? 
https://github.com/Azure/azure-rest-api-specs/blob/a16488bbd0d17551c729e22ea6ac6f7c6588a0a3/specification/common-types/resource-management/v2/types.json#L445

Reply:
this is a new attr we added. will leave it this was for now. will update once support for this is added in the automated swagger generator in  SQL

Reply:
Why not change ref SystemData in commontype now? as there is some diff between your System Data and common type's
for enum to use x-ms-enum. if you are going to change in the future. it will cause breaking change

Reply:
@akning-ms 
can we possibly have a copy of  SystemData from global common type to https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/common/v1/types.json ? It would be easy for us to auto-generate API specs. 

In the APIs of SQL, we use the sql specific common type reference. and there are differences to the global comment type file.

Reply:
Yes. having a copy systemdata from common type v1 is fine for now